demos/tests/cases.py

```diff
@@ -158,6 +158,17 @@ def single_option_cases(key, *args):
         }),
     )),
 
+    CppDemo(name='face_detection_mtcnn_demo', implementation='cpp_gapi',
+            device_keys=['-d_p', '-d_o'],
```

Comment:
```suggestion
            device_keys=['-d_p', '-d_r', '-d_o'],
```

Reply:
Yes, thanks. Can mtcnn composite network work with different devices? OMZ demo has two cases: all on CPU and all on GPU.  Does different devices make sense?

Also OMZ demo has bad performance for case p-net and r-net on GPU.

Reply:
The OMZ demo has any case, but for testing uses only one device for all models.

Reply:
I think it hasn't. [Only all CPU or all GPU](https://github.com/openvinotoolkit/open_model_zoo/blob/b15ae77b045eaa596c3ec453117bc37e3420a5d8/demos/face_detection_mtcnn_demo/python/face_detection_mtcnn_demo.py#L64)